command/ca/admin/admin.go

```diff
@@ -0,0 +1,153 @@
+package admin
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/smallstep/certificates/ca"
+	"github.com/smallstep/certificates/linkedca"
+	"github.com/smallstep/cli/errs"
+	"github.com/smallstep/cli/ui"
+	"github.com/urfave/cli"
+)
+
+// Command returns the jwk subcommand.
+func Command() cli.Command {
+	return cli.Command{
+		Name:      "admin",
+		Usage:     "create and manage the certificate authority admins",
+		UsageText: "step ca admin <subcommand> [arguments] [global-flags] [subcommand-flags]",
+		Subcommands: cli.Commands{
+			listCommand(),
+			//getCommand(),
+			addCommand(),
+			removeCommand(),
+			updateCommand(),
+		},
+		Description: `The **step ca admin** command group provides facilities for managing the
+certificate authority admins.
+
+A admin is an entity that manages administrative resources within a certificate
+authority. Admins manage
+
+* certificate authority configuration
+* provisioner configuration
+* other admins and admin privileges
+
+## EXAMPLES
+
+List the active admins:
+'''
+$ step ca admin list
+'''
+
+Add an admin:
+'''
+$ step ca admin add max@smallstep.com my-jwk-provisioner
```

Comment:
It's the other way around, but perhaps this way looks better.

To decide the order we can try to say it as a phrase:
* Add max@smallstep.com on the my-jwk-provisioner as an admin.
* Add into my-jwk-provisioner, max@smallstep.com as an admin

Reply:
I think I like the first way better. you?